docs/hypernode-deploy/applications/config-for-nextjs.md

```diff
@@ -0,0 +1,129 @@
+# Config for NextJS
+
+```{note}
+NextJS is not officially supported on the Hypernode Platform. So please use this at your own risk,.
+```
+
+Configuration to use as Hypernode Deploy deploy.php for a NextJS webserver application:
+
+```php
+<?php
+
+declare(strict_types=1);
+
+namespace Hypernode\DeployConfiguration;
+
+use function Deployer\run;
+use function Deployer\task;
+use function Deployer\test;
+use function Deployer\within;
+
+task('node:install', static function () {
+    run('npm ci');
+});
+
+task('node:build', static function () {
+    run('npm run build');
+});
+
+task('deploy:pm2:install', static function() {
+    if (!test('grep -q "/data/web/.local/bin" ~/.bashrc')) {
+        run('echo "export PATH=/data/web/.local/bin:$PATH" >> ~/.bashrc');
+        run('source ~/.bashrc');
+    }
+
+    run('npm install -g pm2 --prefix /data/web/.local');
+});
+
+task('deploy:pm2:restart', static function() {
+    within('{{release_path}}', function () {
+        run('pm2 startOrRestart ecosystem.config.js --env prod');
+    });
+});
+
+$configuration = new Configuration();
+
+$configuration->setPlatformConfigurations([
+    new PlatformConfiguration\NginxConfiguration('etc/nginx'),
+    new PlatformConfiguration\HypernodeSettingConfiguration('supervisor_enabled', 'False'),
+    new PlatformConfiguration\HypernodeSettingConfiguration('rabbitmq_enabled', 'False'),
+    new PlatformConfiguration\HypernodeSettingConfiguration('elasticsearch_enabled', 'False'),
+    new PlatformConfiguration\HypernodeSettingConfiguration('opensearch_enabled', 'False'),
+    new PlatformConfiguration\HypernodeSettingConfiguration('varnish_enabled', 'False'),
+    new PlatformConfiguration\HypernodeSettingConfiguration('nodejs_version', '20'),
+]);
+
+$configuration->addBuildTask('node:env');
```

Comment:
I'm missing the registration of the `node:env` task here. Or is it a builtin of Deployer?